File: Company Profile/TugasAkhir/ZoomV1/app/src/main/java/live/hms/zoomv1/network/ApiUtils.java
package live.hms.zoomv1.network;

public final class ApiUtils {

    public static BaseApiService getBase() {
        return RetrofitClient.getClient(ApiConfig.URL).create(BaseApiService.class);
    }

    public static BaseApiService getBaseFCM() {
        return RetrofitClient.getClient(ApiConfig.URL_FIREBASE).create(BaseApiService.class);
    }
}

File: Jet Pack Compose/Atokoku/app/src/main/java/com/apps/atokoku/network/rest/ApiUtils.kt
package com.apps.atokoku.network.rest

import com.apps.atokoku.util.ApiConfig

class ApiUtils {
    companion object {

        fun getBase(): BaseApiService {
            return RetrofitClient.getClient(ApiConfig.URL).create(BaseApiService::class.java)
        }

    }
}